### PR TITLE
Feat(eos_cli_config_gen): Support bpduguard rate-limit

### PR DESCRIPTION
**Pull request by tgodaA**
## Change Summary

Support bpduguard rate-limit knobs

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

```yaml
spanning_tree:
  bpduguard_rate_limit:
    default: < true | false >
    count: < integer >
```

## How to test

Molecule

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md [hunk 1]
```diff
@@ -3290,6 +3290,9 @@ spanning_tree:
     bpdufilter_default: < true | false >
     bpduguard_default: < true | false >
   mode: < mstp | rstp | rapid-pvst | none >
+  bpduguard_rate_limit:
+    default: < true | false >
+    count: < integer >
```
Comment (ankudinov) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md:3296]: Please add some clarification explaining what is this `<integer>`.

**Review by ankudinov — APPROVED** [review]
LGTM. Except minor comment related to count variable.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md (hunk 1 above)
Comment (tgodaA) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md:3296]: Sure

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**